opensrp-giz-malawi/src/main/java/org/smartregister/giz/activity/ChildRegisterActivity.java

```diff
@@ -55,6 +57,7 @@ protected void registerBottomNavigation() {
         }
         bottomNavigationView.getMenu().removeItem(R.id.action_scan_qr);
         bottomNavigationView.getMenu().removeItem(R.id.action_scan_card);
+        bottomNavigationView.setVisibility(View.GONE);
```

Comment:
Is there a way to remove it completely? Instead of hiding it?

Reply:
removing it completely from giz will involve making changes to other modules such as child that use it....NPE issues will come up

Reply:
i just found a way

Reply:
I just discovered this flag https://github.com/OpenSRP/opensrp-client-child/blob/master/opensrp-child/src/main/java/org/smartregister/child/util/ChildAppProperties.java#L16 which is supposed to disable the bottom navigation bar

Reply:
cool

Reply:
But its used like so.....it will still involve changing visibility https://github.com/OpenSRP/opensrp-client-child/blob/d70f0cb9a42bdc8d2175d672fea7ee20f430d85e/opensrp-child/src/main/java/org/smartregister/child/activity/BaseChildRegisterActivity.java#L57